lib/pure/net.nim

```diff
@@ -36,9 +38,17 @@
 ##   var socket = newSocket()
 ##   socket.connect("google.com", Port(80))
 ##
+## For SSL, use the next example.
+##
+## .. code-block:: Nim
+##   var socket = newSocket()
+##   var ctx = newContext()
+##   wrapSocket(ctx, socket)
+##   socket.connect("google.com", Port(443))
+##
 ## UDP is a connectionless protocol, so UDP sockets don't have to explicitly
-## call the ``connect`` procedure. They can simply start sending data
-## immediately.
+## call the `connect<net.html#connect%2CSocket%2Cstring>`_ procedure. They can
```

Comment:
```suggestion
## call the `connect <net.html#connect%2CSocket%2Cstring>`_ procedure. They can
```

Very minor styling change.

We might also want to note that on certain systems (at the very least most POSIX systems), `connect` can be used to bound UDP sockets to a peer, and which will cause all `send/recv` commands after to send/receive only to/from that peer. I've not verified whether this behavior also apply to Nim's `Socket`, so please verify before adding this into the documentation.